src/index.ts

```
@@ -121,6 +135,11 @@ function start() {
   }
 
   continueLink.addEventListener('click', async () => {
+    if (isValidSuspectHref(suspectHref) === false) {
+      console.log(`Disallowed Protocol, cannot continue.`);
```

Will this appear to fail silently to a user?

---

It will fail silently to the user. However, typically, a user will be directed to this view after trying to access an endpoint via `^https?:\/\/`. 

If we have a UX recommendation so that we can not fail silently to the user, I am all ears :D

---

Sorry I'm a bit confused. What will the user see if they hit this condition? -> The phishing page stays put, and as they click the `continue at your own risk` button, they see no indication as to why the button appears to not work?

---

Piggybacking on Alex's request a bit, what's the context that prompted this change? Why would we not want to redirect in this case? I'm also wondering if perhaps in this case instead of having a button that doesn't do anything, perhaps we should use some different copy that explains what's special about this particular URL and then we just _don't_ show a button.

---

> I'm also wondering if perhaps in this case instead of having a button that doesn't do anything, perhaps we should use some different copy that explains what's special about this particular URL and then we just don't show a button.

Yea this is what I'm thinking as well

---

LGTM